package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reverbdotcom/url-template",
-  "version": "2.0.11",
+  "version": "2.1",
```

Comment:
I figure if we were in the `2.0.x` before, `2.1` is probably the move here since this is a larger change?